### ISSUE TITLE
Buttons don't seem to be working

### ISSUE THREAD

**Issue by Notsalgia** [dankgrinder/dankgrinder#109]
ERRO[0010] await resume timed out for: pls pm            cluster=default instance="Loco#9213"
ERRO[0019] await resume timed out for: pls search        cluster=default instance="Loco#9213"
ERRO[0027] await resume timed out for: pls hl            cluster=default instance="Loco#9213"
ERRO[0036] await resume timed out for: pls fish          cluster=default instance="Loco#9213"
ERRO[0059] await resume timed out for: pls search        cluster=default instance="Loco#9213"
ERRO[0068] await resume timed out for: pls hl            cluster=default instance="Loco#9213"

Im getting this error for everything, its like the program is unable to press buttons... anyone got the same problem?

**Comment by Notsalgia** [comment]
Nevermind I found the other fork with working buttons, is there any way to run it in repl.it though?

**Comment by FlickNoJutsu** [comment]
> Nevermind I found the other fork with working buttons, is there any way to run it in repl.it though?

No you cannot

**Comment by TheFinality** [comment]
what other fork?

**Comment by ghost** [comment]
> Nevermind I found the other fork with working buttons, is there any way to run it in repl.it though?

No, you cannot. Dankgrinder requires Go v1.15, replit currently runs Go v1.14. I am making [my own version](https://github.com/didlly/grank) of the code in Python which can be supported by replit.

**Comment by ghost** [comment]
> what other fork?

[This](https://github.com/V4NSH4J/dankgrinder) fork.
